packages/web-components/tests/e2e-storybook/cypress/integration/card/card-static.e2e.js

```diff
@@ -0,0 +1,139 @@
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Defines the component variant path.
+ *
+ * @type {string}
+ * @private
+ */
+const _path = '/iframe.html?id=components-card--static';
+
+/**
+ * Defines the base card component selector.
+ *
+ * @type {string}
+ * @private
+ */
+const _selectorBase = `[data-autoid="${ddsPrefix}--card"]`;
+
+/**
+ * Defines the card element selectors.
+ *
+ * @type {Object.<string>}
+ * @private
+ */
+const _selectors = {
+  eyebrow: `${_selectorBase} [data-autoid="${ddsPrefix}--card-eyebrow"]`,
+  heading: `${_selectorBase} [data-autoid="${ddsPrefix}--card-heading"]`,
+  footer: `${_selectorBase} [data-autoid="${ddsPrefix}--card-footer"]`,
+  image: `${_selectorBase} [data-autoid="${ddsPrefix}--image"]`,
+  tagGroup: `${_selectorBase} [data-autoid="${ddsPrefix}--tag-group"]`,
+  copy: `${_selectorBase} p`,
+};
+
+/**
+ * Defines viewport dimensions to be used in tests.
+ *
+ * @type {number[]}
+ * @private
+ */
+const _viewportDimensions = [1280, 780];
+
+describe('dds-card | static', () => {
+  beforeEach(() => {
+    cy.viewport(..._viewportDimensions);
+    cy.visit(_path);
+  });
+
+  it('should render eyebrow content', () => {
+    cy.get(_selectors.eyebrow).should('not.be.empty');
+  });
+
+  it('should render heading content', () => {
+    cy.get(_selectors.heading).should('not.be.empty');
+  });
+
+  it('should render footer content', () => {
+    cy.get(_selectors.footer).should('not.be.empty');
+  });
+
+  it('should render copy', () => {
+    cy.get(_selectors.copy).each($p => {
+      expect($p).to.not.be.empty;
+    });
+  });
+
+  it('should not respond to a click ', () => {
+    let initialLocation;
+    cy.location('href')
+      .then(location => {
+        initialLocation = location;
+      })
+      .get(_selectorBase)
+      .click()
+      .location('href')
+      .then(location => {
+        expect(location).to.equal(initialLocation);
+      });
+  });
+
+  it('should render correctly in all themes', () => {
+    cy.wrap(['w', 'g10', 'g90', 'g100']).each(theme => {
+      const screenshotTitle = `${Cypress.currentTest.titlePath.join(' | ')} [${theme.toUpperCase()}]`;
+      cy.get('html')
+        .then(doc => doc.attr('storybook-carbon-theme', theme))
+        .screenshot(screenshotTitle);
+      // .percySnapshot(screenshotTitle);
+    });
+  });
+});
+
+describe('dds-card | static with outline', () => {
+  beforeEach(() => {
+    cy.viewport(..._viewportDimensions);
+    cy.visit(`${_path}&knob-Outlined%20card_Card=true`);
+  });
+
+  it('should render with a border', () => {
+    cy.get(_selectorBase).should('have.attr', 'border');
+    cy.get(_selectorBase).should('have.css', 'border');
+  });
+});
+
+describe('dds-card | static with tags', () => {
+  beforeEach(() => {
+    cy.viewport(..._viewportDimensions);
+    cy.visit(`${_path}&knob-Add%20tags_Card=true`);
+  });
+
+  it('should render tags', () => {
+    cy.get(_selectors.tagGroup).should('have.length', 1);
+  });
```

Comment:
@ariellalgilmore Here's a test I already have for tags. Is this sufficient?

Reply:
yeah let's just add a `cy.screenshot` and a commented out percy snapshot like

> // Take a snapshot for visual diffing
  // TODO: click states currently not working in percy for web components
    // cy.percySnapshot('dds-card | static with tags', {
    //   widths: [1280],
    // });